aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -256,6 +256,45 @@ In Visual Studio, the default launch profile is for IIS Express. To run the proj
 
 For apps hosted by HTTP.sys that interact with requests from the Internet or a corporate network, additional configuration might be required when hosting behind proxy servers and load balancers. For more information, see [Configure ASP.NET Core to work with proxy servers and load balancers](xref:host-and-deploy/proxy-load-balancer).
 
+## Advanced HTTP/2 features to support gRPC
+
+Additional HTTP/2 features in HTTP.sys support gRPC, including support for response trailers and sending reset frames.
+
+Requirements to run gRPC with HTTP.SYS:
+
+* Windows 10 Version 2004, OS Build 20300.1000 or later
```

Comment:
This has worked with HttpSys a lot longer than it's worked in IIS. I think it's 20H1 (2004), who's public build number is 19041.508.

Reply:
Oops, was going to tag you in this for the build number.